src/sentry/lang/javascript/processing.py

```diff
@@ -222,18 +221,6 @@ def generate_scraping_config(project: Project) -> Dict[str, Any]:
     }
 
 
-def _normalize_frame(frame: Any) -> dict:
-    frame = dict(frame)
-
-    if abs_path := frame.get("abs_path"):
-        frame["abs_path"] = unquote(abs_path)
-
-    # Symbolicator will *output* `data`, but never use it from the input
-    frame.pop("data", None)
```

Comment:
pls keep the function and this part

Reply:
So you would just remove the `unquote` part?

Reply:
Yes.